crates/subspace-networking/src/node.rs

```diff
@@ -1,4 +1,5 @@
 use crate::shared::{Command, CreatedSubscription, ExactKademliaKey, Shared};
+use crate::Request;
```

Comment:
Try to avoid using imports from other modules, it leads to situations where import in one module is only there because other module imports it, leading to confusion and maintenance inconveniences:
```suggestion
use crate::request_response_handler::Request;
```

Reply:
Fixed here: [74266e9](https://github.com/subspace/subspace/pull/541/commits/74266e94b9393b1ce550d1f0b9439233c1d24895)